docs/sources/enterprise/license-restrictions.md

```diff
@@ -23,9 +27,13 @@ In the context of licensing, each user is classified as either a viewer or an ed
 
 Restrictions are applied separately for viewers and editors.
 
-When the number of maximum active viewers or editors is reached, Grafana displays a warning banner.
+When the number of maximum active viewers or editors is reached, only they can sign in. New users or non-active users cannot sign in.
+
+### Concurrent sessions limit
+
+Sometimes it is useful to sign in to an account from multiple locations simultaneously. As of Grafana Enterprise 7.5.0, accounts are limited to the number of concurrent sessions authorized in your license, which is typically three. A new session is created when you sign in to Grafana from a new device, a different browser, or an incognito window. If you sign in to Grafana from another tab or window within the same browser, then only one session is used.
```

Comment:
```suggestion
Sometimes it is useful to sign in to an account from multiple locations simultaneously. As of Grafana Enterprise 7.5+, accounts are limited to the number of concurrent sessions authorized in your license, which is typically three. A new session is created when you sign in to Grafana from a new device, a different browser, or an incognito window. If a user signs in to Grafana from another tab or window within the same browser, then only one licensed session is used.
```

Reply:
It seems weird to use both `you` and `a user` in two very similar sentences. Also, I'm not sure about "licensed session", what about "user session"?